package.json

```diff
@@ -36,6 +36,7 @@
     "onCommand:quarkusTools.debugQuarkusProject",
     "onCommand:quarkusTools.welcome",
     "workspaceContains:**/src/main/resources/application.properties",
+    "workspaceContains:**/src/main/resources/application.yaml",
     "onLanguage:quarkus-properties",
     "onLanguage:java",
     "onLanguage:qute"
```

Comment:
Should be `"onLanguage:qute-html"`.

If you want to keep `qute`, please change https://github.com/redhat-developer/vscode-quarkus/blob/2bd8d5013ad1bd9f151275af93cd093eb3ad3260/package.json#L66

Reply:
@xorye I did nothing about that, master contains that https://github.com/redhat-developer/vscode-quarkus/blob/master/package.json#L41

I think it should be fixed on an another PR.

Reply:
@angelozerr oops, you are right. I will make a PR for this.

Reply:
thanks!